email/content.h

```diff
@@ -45,4 +46,16 @@ struct Content
   bool cr     : 1; ///< Has CR, even when in a CRLF pair
 };
 
+/**
+ * struct ContentState - Info about the body of an email
+ */
+struct ContentState
+{
+  bool from;
+  int whitespace;
+  bool dot;
+  int linelen;
+  bool was_cr;
```

Comment:
This has just been moved not created, so for **optional** extra brownie points, please document the members :heart:

Reply:
Good point, will document.